contribs/drt/src/main/java/org/matsim/contrib/drt/run/DrtConfigGroup.java

```diff
@@ -113,6 +113,27 @@ public static DrtConfigGroup getSingleModeDrtConfig(Config config) {
 	@PositiveOrZero
 	public double maxAbsoluteDetour = Double.POSITIVE_INFINITY;// [s]
 
+	@Parameter
+	@Comment(
+		"Defines the maximum allowed absolute detour based on the unsharedRideTime. A linear combination similar to travel time constrain is used"
+			+ "This is the ratio part")
+	@DecimalMin("1.0")
+	public double maxDetourAlpha = Double.POSITIVE_INFINITY;
+
+	@Parameter
+	@Comment(
+		"Defines the maximum allowed absolute detour based on the unsharedRideTime. A linear combination similar to travel time constrain is used"
+			+ "This is the constant part")
+	@PositiveOrZero
+	public double maxDetourBeta = Double.POSITIVE_INFINITY;// [s]
```

Comment:
These two comments should mention that the detour is computed from the latest promised pickup time and not from the actual pickup time.

I think we should also mention the following somewhere (maybe in `MaxDetourInsertionCostCalculator`):
By taking into account the latest promised pickup time (instead of the actual pickup time), we simplify the handing of this constraint, because the actual pickup time remains unknown until the moment of picking up the passenger.

BTW. In a simulation setup without congestion, the actual pickup time is always less than or equal to the promised time. With congestion there is no such a guarantee.

Reply:
Thanks, @michalmac! I will update the descriptions and comments based on the comments.